src/game/net.rs

```diff
@@ -75,4 +76,11 @@ impl TwoHeadedNetBase {
 
         return (val, probs);
     }
+
+    pub fn softmax_normalizatione(x: Vec<f32>) -> Vec<f32> {
+        let max_p = x.iter().cloned().fold(f32::MIN, f32::max);

```

Comment:
i don't understand where this softmax comes from, i sent you a whatsapp

Reply:
replayed to you on whatsapp :)